docs/SortOrder.md

```diff
@@ -0,0 +1,7 @@
+# SquareConnect::SortOrder
```

Comment:
Not a blocker but strange to have an empty file here. We should investigate the cause of these, maybe related to the orphaned models we need to clean up from `api.json`.